README.md

```diff
@@ -404,6 +404,7 @@ Engines, libraries and other helpful things specifically for making games.
   alternative to other libraries for doing all those things. Various licenses,
   all open source.
 * [Spearmint][371] - Engine designed for FPS games. [``GPL-3.0-or-later``][GPL-3.0-or-later]
+* [tinyc.game][557] - Tiny C games you can compile and run RIGHT NOW http://tinyc.games [``MIT``][MIT]
```

Comment:
Wow, this is awesome! However, I think this is more of a learning resource than anything. 'Reference resources online' seems like the best section for this to go into. Could you move it there please?